_includes/field-description-payer-reference.md

```diff
@@ -0,0 +1,23 @@
+{% assign documentation_section = include.documentation_section %}
+
+{%- if documentation_section == "checkout" -%}
+{%- assign documentation_url = documentation_section | prepend: "/" | append: "/other-features#merchant-identified-payer" -%}
+
+{%- else -%}
+{%- assign documentation_url = documentation_section | prepend: "/" | append: "/other-features#merchant-identified-payer" -%}
+{%- endif -%}
+
+{%- capture text -%}
+The reference to the consumer from the merchant system, like mobile number,
+customer number etc. Used in [Merchant Identified Payer]({{ documentation_url }}).
```

Comment:
Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5744986)